Pokedex_ios/Helpers/NetworkManager.swift

```diff
@@ -10,7 +10,9 @@ import Apollo
 
 class NetworkManager {
     
+    private let pokeApiEndpoint = "https://beta.pokeapi.co/graphql/v1beta"
```

Comment:
Move this to a global constants file (discuss with your coworkers where will be the best package for that as you three have the same issues

Reply:
Fixed in last commit